src/comps/rf.cc

```diff
@@ -0,0 +1,176 @@
+#include <lapack.hh>
+#include <RandBLAS.hh>
+#include <RandLAPACK.hh>
+
+#define USE_QR
+#define COND_CHECK
+#define VERBOSE
+
+namespace RandLAPACK::comps::rf {
+
+template <typename T>
+void RF<T>::rf1(
+    int64_t m,
+    int64_t n,
+    const std::vector<T>& A,
+    int64_t k,
+    std::vector<T>& Q, // n by k
+    bool use_qr
+){
+    printf("HI, I AM REGULAR RF. \n");
+
+    using namespace blas;
+    using namespace lapack;
+
+    // Get the sketching operator Omega
+    std::vector<T> Omega(n * k, 0.0);
+    T* Q_dat = Q.data();
+
+    RF::RS_Obj.call(m, n, A, k, Omega);
```

Comment:
As with my feedback for the QB code, make sure you use ``this->`` appropriately (for the most part it should be used instead of ``RF::`` in this file).